program/lib/password_validator.rb

```diff
@@ -3,4 +3,9 @@
 # - include at least one of these characters: !, @, $, %, &
 
 def valid?(password)
+    if (password.length >=8) && (password =~ /[!@$%&]/)
```

Comment:
I like the use of the use of the regular expression. It is something I wasn't aware of, and makes your code look very clear and efficient.

Reply:
Thank you very much, I appreciate the feedback